cmake/modules/CompilerFlags.cmake

```diff
@@ -66,4 +70,6 @@ else() # MSVC
 
 	set(CMAKE_C_FLAGS_RELEASE "${FALCOSECURITY_LIBS_RELEASE_FLAGS}")
 	set(CMAKE_CXX_FLAGS_RELEASE "${FALCOSECURITY_LIBS_RELEASE_FLAGS}")
+
+	add_definitions(-D_HAS_STD_BYTE=0)
```

Comment:
This is needed to fix the build on Windows https://developercommunity.visualstudio.com/t/error-c2872-byte-ambiguous-symbol/93889#T-N967428

Reply:
Cool :laughing: